src/backend/gitalytics_api/routes/set_active_workspace.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/python3
+# -*- coding=utf-8 -*-
+r"""
+
+"""
+import typing as t
+import fastapi
+import pydantic
+import database.models as dbm
+from database.db import createLocalSession
+from gitalytics_api.cookies import session_from_cookies, EncryptedCookieStorage
+from gitalytics_api.enums import CookieKey
+
+
+router = fastapi.APIRouter()
+
+
+class ResponseModel(pydantic.BaseModel):
+    status: str = "success"
```

Comment:
default-werte gehen?

Reply:
Mal sehen haha

Reply:
Default Werte waren in der Doku, also erwarte ich, dass es geht. Ich teste es nochmal

Reply:
Mit `return None` geht es anscheinend nicht, aber jetzt schon. Da hat sich ein kleiner bug reingeschlichen